README.md

```diff
@@ -2,6 +2,9 @@
 
 [![codecov](https://codecov.io/gh/ucsb-cs156-s22/s22-4pm-courses/branch/main/graph/badge.svg?token=8mmcEy6jrV)](https://codecov.io/gh/ucsb-cs156-s22/s22-4pm-courses)
 
+* Heroku QA Dashboard (4pm-2): https://dashboard.heroku.com/apps/s22-4pm-2-courses-qa
+* Heroku QA Deployment (4pm-2): https://s22-4pm-2-courses-qa.herokuapp.com/
```

Comment:
I just tried accessing this deployment but I don't see a frontend. You might want to set the production flag :)

Reply:
Fixed!